test/integration/install/test_install.py

```diff
@@ -77,6 +167,84 @@ def test_install_tt_specific_version(tt_cmd, tmpdir):
     assert re.search(r"Tarantool CLI version 1.0.0", start_output)
 
 
+@pytest.mark.slow
+def test_install_tt_specific_commit(tt_cmd, tmpdir):
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
+        f.write('tt:\n  app:\n    bin_dir:\n    inc_dir:\n')
+
+    # Install specific tt's commit.
+    install_cmd = [tt_cmd, "--cfg", configPath, "install", "tt", "2df3077"]
```

Comment:
Doesn't this test duplicate what is done in `test_uninstall_tt_specific_commit`? Install and uninstall tests can be merged to avoid duplicate program installation, which is a time consuming operation.

Reply:
fixed